draft-ietf-rats-uccs.md

```diff
@@ -456,6 +456,15 @@ CWT-kid = bytes
 ~~~
 {: sourcecode-name="uccs-additional-examples.cddl"}
 
+Specifications that use UCCS in the context of EAT {{-eat}} need the glue CDDL defined in {{fig-uccs-eat-glue}}.
+
+~~~ cddl
+$JSON-Selector-Type /= "UCCS"
+$JSON-Selector-Value /= ~UCCS
+$EAT-CBOR-Tagged-Token = UCCS
```

Comment:
```suggestion
$EAT-CBOR-Tagged-Token /= UCCS
```

thanks to @carl-wallace for  https://github.com/ietf-rats-wg/draft-ietf-rats-uccs/pull/17#issuecomment-1712129351